### DISCUSSION TITLE
Storing Multiple Records as array

### DISCUSSION THREAD

**Discussion by imran300** [livewire/livewire#4819] [1. Help]
Hello,
I was trying to store multiple records using livewire. 
My Input field is like wire model equals tax_rate[]

I initialized public property as " public $tax_rate = []; 
but it gave me an error. How to achieve this in,livewire? Is their any help in documentation.

Thanks
![Capture](https://user-images.githubusercontent.com/7050400/158947448-7458939d-db9b-46bb-9a62-8f7703afe5a0.PNG)

**Comment by zepfietje** [comment]
Hey @imran300!

Could you share some more details like a snippet of your code?
This makes it easier for us to help, thanks!

**Reply by imran300** [reply]
Hello,
Thank Your for your kind response. 
here are the screenshots
![Capture](https://user-images.githubusercontent.com/7050400/159170982-aa0ca8f6-23bc-4bd6-9577-894c83bce1b0.PNG)
![1](https://user-images.githubusercontent.com/7050400/159170984-308c1c1f-99dd-415f-9f0e-37b372aaec8d.PNG)


my input name would be an array and i will simple loop through it and it will give me indexes of each input ...
Can achieve such think in Livewire as Livewire doesnt allow array properties like this.
